### DISCUSSION TITLE
[Feature] which package is SkywalkingActiveSpan in?

### DISCUSSION THREAD

**Discussion by qinshaokang** [apache/skywalking#8056] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

I want use SkywalkingActiveSpan in my project ,which package is SkywalkingActiveSpan in?

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Could you be more clear where do you read this? And what is your agenda?

I am a little confused.

**Comment by kezhenxu94** [comment]
> I want use SkywalkingActiveSpan in my project ,which package is SkywalkingActiveSpan in?

Learn how to search. It’s in https://github.com/apache/skywalking-java/blob/main/apm-application-toolkit/apm-toolkit-opentracing/src/main/java/org/apache/skywalking/apm/toolkit/opentracing/SkywalkingActiveSpan.java

if you mean maven module, it’s 

```xml
        <artifactId>apm-toolkit-opentracing</artifactId>
        <groupId>org.apache.skywalking</groupId>
```
